src/CommunityToolkit.Maui/AppThemeColor.shared.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// Represents a color that is aware of the operating system theme.
+/// </summary>
+public class AppThemeColor : AppThemeObject<Color>
```

Comment:
Can we make this `sealed`, can't see any value allowing users to inherit from it

```suggestion
public sealed class AppThemeColor : AppThemeObject<Color>
```